packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,479 @@
+
+### Northstar property mapping
+
+#### Props which are repeating in each component
+
+These props are the same in all the components.
+`as` prop had different defaults in different components.
+BreadcrumbDivider has default `span`. BreadcrumbLink has `a` and Breadcrumb has `nav`.
+
+- `accessibility` => Accessibility behavior if overridden by the user.
+- `as` => type React.ElementType - An element type to render as (string or component).
+- `className`
+- `content` => Shorthand for primary content.
+- `design` => type ComponentDesignProp
+- `styles` =>
+- `variables` => Override for theme site variables to allow modifications of component styling via themes.
+
+#### Property Mapping
+
+| Northstar `Breadcrumb` | v9 `Breadcrumb` |
+| ---------------------- | --------------- |
+| `accessibility`        |                 |
+| `as`                   | `as`            |
+| `className`            |                 |
+| `content`              |                 |
+| `design`               |                 |
+| `styles`               |                 |
+| `variables`            |                 |
+
+#### Breadcrumb component:
+
+- `size` => use as it is.
 
-- _Migration from v8_
-- _Migration from v0_
+#### BreadcrumbItem component:
+
+- `active` => Indicates if the link is the active.
+- `disabled` => The Breadcrumb Item can be disabled.
+
+#### Property Mapping
+
+| Northstar `BreadcrumbItem` | v9 `BreadcrumbItem` |
+| -------------------------- | ------------------- |
+| `active`                   | `active`            |
+| `disabled`                 | `disabled`          |
 
 ## Behaviors
 
-_Explain how the component will behave in use, including:_
+#### Overflow
+
+The default position of ellipses should be the second element because from a UX perspective root folder should be shown.
+
+#### Collapse functionality:
+
+Collapse turns on when:
+
+- There's not enough space
+- When `maxDisplayedItems` prop is provided and number of items is bigger than `maxDisplayedItems`.
+
+#### Truncate long names:
+
+This value should be customizable
+
+### Mouse
+
+#### Breadcrumb item as Button
+
+![Breadcrumb item as Button](./assets/button-beadcrumb.png)
+
+#### Breadcrumb item as Link
+
+![Breadcrumb item as Link](./assets/link-beadcrumb.png)
+
+Breadcrumb can have the folloing states:
 
-- _Component States_
-- _Interaction_
-  - _Keyboard_
-  - _Cursor_
-  - _Touch_
-  - _Screen readers_
+- Rest
+- Hover
+- Pressed
+- Selected
+- Focused
+- Disabled
+- Active
+
+![Breadcrumb states](./assets/beadcrumb-states.png)
+
+#### Non-interactive Breadcrumb Item
+
+Non-interactive style variation for places where the Breadcrumb is purely representational or informational. Usually this instance is mostly used to describe file path location, etc.
+![Non-interactive Breadcrumb](./assets/breadcrumb-not-interactive.png)
+
+#### Tooltip
+
+Tooltip is shown `onHover` on collapsed menu or items with long names.
+![Breadcrumb Tooltip](./assets/breadcrumb-tooltip.png)
```

Comment:
I suggest to follow up with @smhigley about this, as there was a RFC related to this topic (#18029).